rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -26,8 +26,8 @@
                 <hint issue-display-mode="all" title="Replace service-style deployments" effort="1" category-id="optional">
                     <message>
                         <![CDATA[
-                        MBeans were part of the core architecture in previous versions of Red Hat JBoss Enterprise Application Platform.  
-                        JBoss Service Archive (SAR) deployments using the JBoss specific `jboss-service.xml` and `jboss-beans.xml` service-style descriptors were used by the application server to create MBeans based on JBoss Beans.  
+                        MBeans were part of the core architecture in previous versions of Red Hat JBoss Enterprise Application Platform.
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.